mypy/semanal.py

```diff
@@ -872,7 +856,8 @@ def get_tvars(self, tp: Type) -> List[Tuple[str, TypeVarExpr]]:
                 tvars.extend(self.get_tvars(arg))
         return self.remove_dups(tvars)
 
-    def remove_dups(self, tvars: List[T]) -> List[T]:
+    @staticmethod
```

Comment:
This should be a module-level method (see above), and most new static methods are probably similar.